.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,30 @@
+---
+name: Integation tests
+"on":
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  maven-it:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        java: [8, 11, 13]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+      - uses: actions/cache@v1
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-jdk-${{ matrix.java }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-jdk-${{ matrix.java }}-maven-
+      - name: Build it with Maven
+        run: mvn clean install -Pitcase
```

Comment:
@olenagerasimova could you please explain why are you using `install` here instead of `verify` or `test` which are usually used to execute tests?

Reply:
@olegmoz `verify` includes `qulice` checks by default, we have this check in another action and I think we can skip it here, `test` command runs only unit tests and specifying profiles (any profile actually) does not seem to work. 

Let me disable qulice for `verify` manually to avoid misunderstanding in the future.